src/api/operator/utils.ts

```diff
@@ -0,0 +1,54 @@
+// Util functions that only pertain to/deal with operator API related stuff
+
+import BigNumber from 'bignumber.js'
+
+import { ONE_BIG_NUMBER } from 'const'
+
+import { RawOrder } from './types'
+
+export type OrderStatus = 'open' | 'filled' | 'expired' | 'partially filled'
+
+// TODO: move to const or app settings?
+const FILLED_ORDER_EPSILON = '0.0001' // == 0.01%
+
+function isOrderFilled(order: RawOrder): boolean {
+  let amount, executedAmount
+
+  if (order.kind === 'buy') {
```

Comment:
do we have an `OrderType` enum somewhere we can use instead of string?

Reply:
I think we do. If it's not exported in the NPM package of the contracts, we should have the enum, or at least some constants

Reply:
Well, it's a type https://github.com/gnosis/gp-ui/blob/develop/src/api/operator/types.ts#L16

I feel like we had this discussion of types vs enums many months ago

Reply:
aha, okok, it would fail if it's anything different than buy, ok, then I'm fine with it

Reply:
Like the other function, this also has auto-complete and TS knows it refers to a type, not a string:
![screenshot_2021-02-01_13-56-53](https://user-images.githubusercontent.com/43217/106523122-84738500-6495-11eb-87c0-65bc46b10809.png)